// nwglobal/index.js
module.exports.Array = Array;
module.exports.Boolean = Boolean;
module.exports.Date = Date;
module.exports.Function = Function;
module.exports.Number = Number;
module.exports.Object = Object;
module.exports.RegExp = RegExp;
module.exports.String = String;
module.exports.ArrayBuffer = ArrayBuffer;
module.exports.DataView = DataView;
module.exports.Float32Array = Float32Array;
module.exports.Float64Array = Float64Array;
module.exports.Int16Array = Int16Array;
module.exports.Int32Array = Int32Array;
module.exports.Int8Array = Int8Array;
module.exports.Uint16Array = Uint16Array;
module.exports.Uint32Array = Uint32Array;
module.exports.Uint8Array = Uint8Array;
module.exports.Error = Error;
module.exports.EvalError = EvalError;
module.exports.RangeError = RangeError;
module.exports.ReferenceError = ReferenceError;
module.exports.SyntaxError = SyntaxError;
module.exports.TypeError = TypeError;
module.exports.URIError = URIError;
module.exports.Infinity = Infinity;
module.exports.NaN = NaN;
module.exports.undefined = void 0;
module.exports.null = null;
